core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -274,14 +282,17 @@ public FileInfo getFileInfo(String ufsFullPath, GetStatusPOptions options) throw
     if (status == null) {
       // The requested FileStatus is not present in memory cache.
       // Let's try to query local persistent DoraMetaStore.
-      Optional<DoraMeta.FileStatus> fs = mMetaStore.getDoraMeta(ufsFullPath);
+      Optional<DoraMeta.FileStatus> fs = mMetaStore != null ? mMetaStore.getDoraMeta(ufsFullPath) :
```

Comment:
one minor suggestion is can we help improve the logics of this method to make it more clear?

Reply:
Refactored for better understanding. Please take another look.